app/src/main/java/fr/free/nrw/commons/media/zoomControllers/zoomable/ZoomableDraweeView.java

```diff
@@ -73,9 +74,18 @@ public void onTransformChanged(Matrix transform) {
                 }
 
                 @Override
-                public void onTransformEnd(Matrix transform) {}
+                public void onTransformEnd(Matrix transform) {
+                    if (null != transformationListener) {
+                        transformationListener.onTransformationEnd();
+                    }
+                }
             };
 
+    public void setTransformationListener(
```

Comment:
was this too long to fit on 1 line?

Reply:
Actually yes :P

Reply:
Ohh, sorry, do you mean trying the if, without the braces?

Reply:
Ohh, sorry, do you mean trying the if, without the braces?

Reply:
no, I meant this line. you could shorten the parameter name but it is not necessary